imports/plugins/included/marketplace/server/no-meteor/util/stripeListRefunds.js

```diff
@@ -25,11 +25,13 @@ export default async function stripeListRefunds(context, paymentMethod) {
   if (refundListResults && refundListResults.data) {
     for (const refund of refundListResults.data) {
       result.push({
-        type: refund.object,
+        _id: refund.id,
         amount: refund.amount / 100,
```

Comment:
I know this line didn't change but are we USD-only in stripe at this point? Is it safe to do this conversion regardless of `refund.currency`?

Reply:
Great question. It's probably not safe to do in all currencies: https://stripe.com/docs/currencies#zero-decimal

@spencern are we USD-only for Stripe?

Reply:
We should not assume USD only.

@kieckhafer will you create a ticket to audit the stripe integration to see where we have incompatibility with other currencies?

Reply:
Issue created: https://github.com/reactioncommerce/reaction/issues/5383